app/assets/javascripts/js-cookie/js.cookie.js

```diff
@@ -0,0 +1,145 @@
+/*!
+ * JavaScript Cookie v2.0.4
+ * https://github.com/js-cookie/js-cookie
+ *
+ * Copyright 2006, 2015 Klaus Hartl & Fagner Brack
+ * Released under the MIT license
+ */
+(function (factory) {
+	if (typeof define === 'function' && define.amd) {
+		define(factory);
```

Comment:
@houndci yeah, that's why the code is checking if `define` is defined, with `typeof`.